### PR TITLE
Create regions.md

### PR DESCRIPTION
**Pull request by danieltprice**
Add regions content.
https://websitemain62807-dpriceregionsupportupdates.gtsb.io/docs/conceptual-guides/regions/

### CODE REVIEW

File: content/docs/get-started-with-neon/regions.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+enableTableOfContents: true
+isDraft: true
+---
+Neon supports project deployment in several regions. We recommended that you select the region closest to your application server to reduce latency between your Neon project and your application.
+
+## Available regions
+
+Neon currently supports the following AWS regions:
+
+- Asia Pacific (Singapore) Region &mdash; `ap-southeast-1`
```
Comment (zoete) [inline content/docs/get-started-with-neon/regions.md:12]: Currently US is more important to us than APAC, we got to list them in first place.

File: content/docs/get-started-with-neon/regions.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+## Moving project data to a new region
+
+The region for an existing Neon project cannot be changed.
+
+If you need to move your data to a different region, the following steps are recommended:
+
+1. Create a new project in the new region. For project creation instructions, see [Setting up a project](../setting-up-a-project).
+1. Dump and restore data from your old project to the new project. For instructions, see [Import data from PostgreSQL](../../how-to-guides/import-an-existing-database).
+
```
Comment (zoete) [inline content/docs/get-started-with-neon/regions.md:32]: Could we add a section at the end saying something like "let us know what regions you are looking forward to on our community forum"?

**Review by zoete — CHANGES_REQUESTED** [review]
One comment, one suggestion

File: content/docs/get-started-with-neon/regions.md (hunk 2 above)
Comment (danieltprice) [reply content/docs/get-started-with-neon/regions.md:32]: Added!

File: content/docs/get-started-with-neon/regions.md (hunk 1 above)
Comment (danieltprice) [reply content/docs/get-started-with-neon/regions.md]: Reordered as suggested.

**Review by zoete — DISMISSED** [review]

File: content/docs/sidebar.yaml [hunk 3]
```diff
@@ -18,6 +18,8 @@
       slug: get-started-with-neon/signing-up
     - title: Setting up a project
       slug: get-started-with-neon/setting-up-a-project
+    - title: Regions
```
Comment (stepashka) [inline content/docs/sidebar.yaml:21]: not sure about the placement
i suggest to consider mentioning region in the create a project / set up a project page in the docs
also add regios to glossary?
once that is done not sure if we need a separate getting started guide on this topic

Comment (stepashka) [inline content/docs/sidebar.yaml:21]: not sure about the placement
i suggest to consider mentioning region in the create a project / set up a project page in the docs
also add regios to glossary?
once that is done not sure if we need a separate getting started guide on this topic

Comment (danieltprice) [reply content/docs/sidebar.yaml:23]: Moved under the Introduction section, similar to what PlanetScale does.
PR created for region definition: https://github.com/neondatabase/website/pull/283

**Review by zoete — APPROVED** [review]

**Comment by zoete** [comment]
Merge this once https://github.com/neondatabase/cloud/pull/2767 is merged!

**Pull request merged**